### PR TITLE
doc: Fixing the broken hyperlinks by pointing to correct documentation.

### PR DESCRIPTION
**Pull request by umullan**
Signed-off-by: umullangi@walmartlabs.com

### CODE REVIEW

File: doc/rbd/rbd-openstack.rst [hunk 1]
```diff
@@ -176,7 +176,7 @@ Save the uuid of the secret for configuring ``nova-compute`` later.
    However from a platform consistency perspective, it's better to keep the
    same UUID.
 
-.. _cephx authentication: ../../rados/operations/authentication
+.. _cephx authentication: ../../rados/operations/user-management/#add-a-user
```
Comment (tchaikov) [inline doc/rbd/rbd-openstack.rst]: probably we can point this reference to ../../rados/configuration/auth-config-ref#enabling-disabling-cephx instead? which i think is more relevant?

File: doc/radosgw/federated-config.rst [hunk 2]
```diff
@@ -817,7 +817,7 @@ there is a unified namespace between the two regions.
 
 .. _CRUSH Map: ../../rados/operations/crush-map
 .. _Install Ceph Object Gateway: ../../install/install-ceph-gateway
-.. _Cephx Administration: ../../rados/operations/authentication/#cephx-administration
+.. _Cephx Administration: ../../rados/operations/user-management/#managing-users
```
Comment (tchaikov) [inline doc/radosgw/federated-config.rst]: we used to have a paragraph referencing this section
```rst
When accessing the Ceph Storage Cluster, each instance of a Ceph Object Gateway
must provide the Ceph Storage Cluster with a user name and key. We recommend
setting up at least one user name and key for each zone. See `Cephx
Administration`_ for a discussion on adding keyrings and keys.
```
this paragraph is now a part of "Create a Keyring", and the link was removed. maybe we can remove the reference as well?

File: doc/install/upgrading-ceph.rst [hunk 3]
```diff
@@ -742,8 +742,8 @@ Under those directories, the keyring should be in a file named ``keyring``.
 
 .. _Monitor Config Reference: ../../rados/configuration/mon-config-ref
 .. _Joao's blog post: http://ceph.com/dev-notes/cephs-new-monitor-changes 
-.. _Ceph Authentication: ../../rados/operations/authentication/
-.. _Ceph Authentication - Backward Compatibility: ../../rados/operations/authentication/#backward-compatibility
+.. _Ceph Authentication: ../../rados/operations/user-management/
```
Comment (tchaikov) [inline doc/install/upgrading-ceph.rst]: might want to update the place referencing this section as well to reflect the updated name of the "Authentication" section. like
```diff
diff --git a/doc/install/upgrading-ceph.rst b/doc/install/upgrading-ceph.rst
index 3ff63e2..b7f8889 100644
--- a/doc/install/upgrading-ceph.rst
+++ b/doc/install/upgrading-ceph.rst
@@ -174,8 +174,8 @@ daemon cannot participate in the new protocol).  We recommend not waiting for an
 extended period of time between ``ceph-mon`` upgrades. See `Upgrading 
 Monitors`_ for details.
 
-.. note:: See the `Authentication`_ section and the 
-   `Ceph Authentication - Backward Compatibility`_ for additional information
+.. note:: See the `User Management`_ section and the 
+   `User Management - Backward Compatibility`_ for additional information
    on authentication backward compatibility settings for Bobtail.
 
 Once you complete the upgrade of your monitors from Argonaut to
```

File: doc/cephfs/kernel.rst [hunk 4]
```diff
@@ -31,4 +31,4 @@ To unmount the Ceph file system, you may use the ``umount`` command. For example
 See `mount.ceph`_ for details.
 
 .. _mount.ceph: ../../man/8/mount.ceph/
-.. _Authentication: ../../rados/operations/authentication/
\ No newline at end of file
+.. _Authentication: ../../rados/operations/user-management/
```
Comment (tchaikov) [inline doc/cephfs/kernel.rst]: might want to update the place referencing this section as well to reflect the updated section name of the "Authentication" section.

File: doc/cephfs/fstab.rst [hunk 5]
```diff
@@ -44,4 +44,4 @@ Ensure you use the ID (e.g., ``admin``, not ``client.admin``). You can pass any
 See `Authentication`_ for details. 
 
 
-.. _Authentication: ../../rados/operations/authentication/
+.. _Authentication: ../../rados/operations/user-management/
```
Comment (tchaikov) [inline doc/cephfs/fstab.rst]: might want to update the place referencing this section as well to reflect the updated section name of the "Authentication" section.

**Comment by umullan** [comment]
Kefu Chai, I have incorporated your comments. Can you please review the changes ?

**Comment by tchaikov** [comment]
lgtm.

**Pull request merged**